docs/docs/glossary/index.md

```diff
@@ -108,3 +108,85 @@ contract Test {
     }
 ]
 ```
+
+## Proxy
```

Comment:
@danforbes what do you suggest for keeping glossary section?

Reply:
Thanks for the feedback, I will work on this